packages/fixie/src/voice.ts

```diff
@@ -108,6 +108,11 @@ export class VoiceSession {
     return this.outAnalyzer?.analyzer;
   }
 
+  /** Returns the Room ID currently in use by this VoiceSession. */
+  get roomId() : string | undefined {
```

Comment:
Technically this is a roomName, livekit rooms also have room IDs but they are different.